src/libstat/classifiers/bayes.c

```diff
@@ -49,7 +49,7 @@ bayes_error_quark (void)
 /**
  * Returns probability of chisquare > value with specified number of freedom
  * degrees
- * @param value value to test
+ * @param value to test
```

Comment:
Looks like I want to drop the whole value set (I should have re-run the check after I updated my patterns, but I guess I thought I found a case where this change was right).